cli/htmgo/tasks/astgen/entry.go

```diff
@@ -402,6 +414,18 @@ func GetModuleName() string {
 	return modName
 }
 
+func matchProjectDirectory() bool {
+	wd := process.GetWorkingDir()
+	dirs := strings.Split(wd, "/")
+
+	projectDir := dirs[len(dirs)-1]
+
+	if GetModuleName() == projectDir {
```

Comment:
I actually don't believe its required these match, for example: the framework module in htmgo is 
```
github.com/maddalax/htmgo/framework
```

but the project dir path would be just `framework`

which would make this fail. Can we remove this check?

Reply:
so the project directory check is not required anymore or should we use a different approach?

Reply:
ah I see, sorry I didn't explain it at all in the GH issue. By that I meant make sure basically that there is:

1. go.mod
2. pages directory
3. partials directory

basically ensure its a valid project structure

Reply:
Ohhh gotcha, let me implement that then, thanks for clarifying